cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final Locale DE = Locale.forLanguageTag("de");
+    private static final Locale EN_US = Locale.forLanguageTag("en-US");
+    private static final List<Locale> SUPPORTED_LOCALES = asList(DE, EN_US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
```

Comment:
I think we should not expose these credentials, you could pass them as environment variables.
Either you use the travis cli to add them encrypted to .travis.yml (https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables) or enter it directly in travis (https://travis-ci.org/commercetools/commercetools-sunrise-java-cms/settings). If you can't access it, I would add the variables for you.